packages/@react-spectrum/inlinealert/intl/en-US.json

```diff
@@ -0,0 +1,7 @@
+{
+  "neutral": "Neutral",
+  "info": "Informative",
+  "positive": "Positive",
+  "notice": "Notice",
+  "negative": "Negative"
+}
```

Comment:
I think we should be able to copy most of the translations from `@react-spectrum/alert` too

Reply:
I can handle both of these in a follow-up